### ISSUE TITLE
Respiration-related oscillations: Skyra, Prisma (VDx)

### ISSUE THREAD

**Issue by BenInglis** [CMRR-C2P/MB#132]
This is just to bring the CMRR folks into an ongoing conversation begun here:

https://mvpa.blogspot.co.uk/2016/08/thats-motion-respiration.html

So far there is compelling evidence to link a periodic oscillation (translation in-plane) to chest motion via modulation of on-resonance frequency. Data from a Prisma-Fit at WashU (running R013 on what I assume was VDx before August, waiting for confirmation) and all the MyConnectome data collected by Russ Poldrack (Skyra at UT-Austin, data acquired between Oct 2012 & Mar 2014, presumably running VDx, problem noted by Jonathan Power in http://www.sciencedirect.com/science/article/pii/S1053811916303871) exhibit the same phenomenon. 

No testing or information yet on other scanners or for VEx versions of R014. I've asked WashU to test. My working assumption is that this is related to the per volume frequency adjustment that was required in VD on Skyra to offset the drift caused by heating. So I've also asked them to try with and without the new "Disable freq. update" option. I don't believe the Prisma will exhibit the same sort of thermal drift as the Skyra anyway, so the adjustment may well be introducing instability. 

I would also be interested to test (I don't have a Skyra) whether the Siemens product EPI with frequency adjustment shows the same problem.

**Comment by BenInglis** [comment]
From a variety of reports, including from GE scanner owners, it appears that the respiratory oscillations are most likely due to some sort of unintended consequence of the on-resonance frequency feedback that some scanners use to offset high thermal drifts. Thus, it's not an MB-specific issue. However, to my knowledge, only the latest version (R014) of MB-EPI has the capacity to turn the frequency feedback on/off at will. I would be very interested to hear of comparisons with the feedback on vs off. The respiratory oscillations may be most visible on a Skyra, because of the thermal drift, but it would be most useful to hear from people testing on Skyra, Verio and Prisma scanners.

**Comment by jpyles** [comment]
Ben: I can try to run this test on our Verio in the coming weeks. Do you have a recommended protocol/parameters to match up with your testing?

**Comment by eauerbach** [comment]
Note that the "Disable freq. update" option just disables the transmitter frequency updates, and that was only ever enabled for VD/VE versions. For all software versions, there has always been a dynamic B0 correction performed as part of the reconstruction (essentially DORK, which was developed by Josef Pfeuffer and Gary Glover, so I'm sure Siemens and GE are doing exactly the same thing).

The frequency update simply takes the drift calculation from DORK and applies it to the transmitter. Any residual drift is still corrected by applying a phase ramp in k-space to shift the image data. The transmitter frequency update ensures that the total drift over the course of a scan is never too large. Normally that doesn't really matter, since it can easily be corrected in the data, but on Skyra the drift can be so large that updating the transmitter is necessary to ensure that e.g. fat saturation pulses remain effective.

So if the DORK correction is not sufficient to remove respiration-induced B0 effects, or if somehow it is making things worse in some cases, I would expect that problem should be the the same on all scanners. I would be surprised if the transmitter frequency update by itself is causing a problem.

Note that at least in the Siemens implementation, the correction is per-volume. B0 fluctuations due to breathing presumably have a Z gradient component. Especially if you are mostly acquiring axial slices, it would be interesting to see what this motion looks like on a slice-by-slice basis (or within some groups of slices along Z). It is possible to do a per-slice correction, but presumably the vendors do not do this because it will be more noisy.

On VD13/VE11 there is also something else called "Tx feedback", which is a hardware feedback loop intended to correct for global magnitude and phase drifts in the transmitter hardware (related to stability of the RF amplifiers and body coil tuning with heating). I don't think this is at issue here, but I just wanted to clarify that "Tx feedback" has a different specific meaning in the Siemens world not to be confused with "Frequency feedback".

**Comment by BenInglis** [comment]
Hi John, I've not done any testing myself yet because I don't have access to either a Skyra or Prisma running VD13 or VE11. So far, only these scanners & s/w combinations have produced examples of the sort shown in the blog post I linked. Am hoping to test at UCSF in the next few weeks. 

I would suggest testing a range of MB, from 3-8, and TR from as short as possible up to 2000 ms. There is a suspicion that shorter TR (usually facilitated by higher MB) make the oscillations more visible, but so far there's no direct implication of MB or TR as being the root cause.

Probably the most useful tests you can run would be using product EPI on either VD13 or VE11. A small number of slices, with TR from 500-2000 ms would explore most of the parameter space as reported for MB-EPI, albeit with reduced brain coverage. (Thicker slices probably won't hurt.) 

Eddie: many thanks for the clarifications, especially on the terminology. So if I'm understanding your description correctly the "transmitter frequency update" in your first paragraph is equivalent to the "Tx feedback" in the last paragraph, and this has only been enabled for VD13 and VE11. If so, this is my prime suspect based on reports I've received so far. (Also implies GE is doing something similar.) Right now I have no reason to suspect the on-res frequency adjustment because the oscillations being reported are generally clearer with short TR, whereas if the Z gradient of B0 were producing slice-wise errors we might expect those to get worse with longer TR.

Final question. From the release notes for R014:

"Protocol option added to disable frequency feedback, e.g. for timing-
dependent studies." 

This option refers to the Tx feedback, correct? The option description is consistent with what you've written here, i.e.:

"Disable freq. update: This option will disable dynamic transmitter frequency updating, which is intended to correct for frequency drift over the course of long scans (caused by e.g. gradient heating). This update adds a 20 ms dead period at the end of every TR. Image drift in the phase encoding direction will still be corrected in post-processing if this option is selected."

So selecting the option disables Tx frequency adjustment *but leaves* the DORK-style on-resonance correction that is common to all versions.

**Comment by xjqian** [comment]
I believe it is only worthwhile to test this on Prisma. As on Skyra, without "Tx feedback", you get obviously increasing MB artifacts (more ghosting over time) over a short period.

**Comment by BenInglis** [comment]
Hi Gordon, agreed, best to leave "Tx feedback" on for Skyra, to combat drive.

All: latest tests show the same respiratory oscillations on a Prisma running VE11B, R014, at WashU. This is with Tx feedback enabled or disabled. So my working hypothesis is shifting towards this being an "undocumented feature" of MB, similar to the respiratory effects that arise in the ACS scans for in-plane GRAPPA. I'm shifting towards testing effects specific to MB, such as the slice cross-talk issue when using an even number of slice packets, as given by (#slices/MB factor). Also testing on my Trio running VB17A. Full update to come, but for the time being the Tx frequency feedback seems not to be the root cause.

**Comment by xjqian** [comment]
Comparing axial vs. sagittal/coronal slice orientation may help give some insight to your above hypothesis of interaction between MB and breathing, assuming "B0 fluctuations due to breathing presumably have a Z gradient component" is true.

**Comment by BenInglis** [comment]
Excellent idea, Gordon. I've asked the WashU folks to test sag vs axial, since sag should leave the phase encoding A-P by default (I think). I acquired a bunch of different runs on my Trio (VB17A, R014) on Weds morning and have sent the data over to WashU so they can use the exact scripts to analyze my data as their data. Will update as soon as I get a report from them.

**Comment by BenInglis** [comment]
Another quick update. Tests with various MB factors, odd vs even number of slice packets, with/without iPAT in-plane or partial Fourier, all on my Trio (VB17A, RO14), showed translations that were perhaps 5-fold lower pk-pk than other sites have reported. There was also very little dependence on any parameter, including MB. However, one major difference is that I use a flip angle below the Ernst angle. I was using 35 deg at TR=1200 ms. HCP, for instance, used TR=720 ms and FA=52deg. This is a big difference in T1 weighting. Other sites likely have FA at or near the Ernst angle. And given the common tactic of reducing TR via increasing MB, it seems likely that an MB-dependence may be mistaken for a TR,FA dependence. Will investigate ASAP, along with axial vs sag orientation dependence.

**Comment by BenInglis** [comment]
Eddie, is the final set of crusher gradients the same as for ep2d_bold? Years ago we worked on trying to eliminate signal from vitreous that manifests in the most superior slice when doing descending slice ordering, because the superior slice is acquired adjacent in time to the most inferior slice, and the vitreous T2 is so long that it is common to see "eyeballs at the top of the head." We were never quite able to suppress the signal, however. I ask because I'm wondering if we are seeing an SSFP-like effect in CSF when going to short TR, as in this paper: http://www.ncbi.nlm.nih.gov/pubmed/11064411. So far, setting the FA to 30 or 65 deg made no significant difference for axial or sag slices for TR=1200 ms. (Axial vs sag also made no difference.) So now I'm starting to think of more subtle effects. I also received yet another report (USC) of clear respiratory oscillations, so it seems to be ubiquitous.

**Comment by eauerbach** [comment]
I did not change the crushers. But in the BOLD sequence you do have the option of enabling RF spoiling; this is not an option in the product.

**Comment by BenInglis** [comment]
I'm seeing consistent shifts along Z whether using axial or sagittal slices, and no flip angle dependence. At this point it doesn't appear to be related to the MB acquisition scheme at all, begging the question why all the reports of respiratory artifacts now and not previously. A simple explanation may be that with regular EPI and TR of 2 sec or more the oscillations were aliased and/or less obvious. Next week I will try to replicate the shifts I've seen with R014 MB-EPI using ep2d_bold on my Trio. I'll use the same spatial resolution and TR, simply reduce the number of slices.

**Comment by BenInglis** [comment]
So it looks like there are two distinct contributions, one greater than the other. The bigger of the two seems to be a direct mechanical motion effect. We were able to largely eliminate this with a custom printed head holder. The other effect then becomes much clearer: a B0 shift that slightly affects slice position for axial slices, and affects distortion in the phase encode dimension for sagittal and coronal scans. The behavior is consistent between MB=6 and product EPI with matching parameters (except total number of slices).

In answer to the question: Why does MB suddenly seem to be highlighting a problem? I think it's to do with the tendency to match the in-plane resolution to the slice thickness, thereby driving the read and frequency-encode gradients to the max, extending echo spacing and hence distortion. (It's rare for someone to do 2 mm isotropic voxels with conventional EPI, yet rather "standard" once one gets a hold of an MB sequence.) This is coupled with the tendency to use faster sampling, TR < 2000 ms, which increases the visibility of oscillations at respiratory frequencies.

In summary, it doesn't seem to be MB per se, but a consequence of the way people are using MB. There may well be tertiary effects that have yet to been disentangled, too, such as spin history effects when using very short TR and Ernst flip angles, but as of now I am confident that MB gets a clean bill of health.

We're still working through our latest test data, and at some point in the next few days I'll put together a blog post summarizing our findings, but we now have an entirely consistent "picture" and are able to make predictions about what will happen given a particular setup (e.g. type of head restraint), slice direction and spatial resolution. I suggest leaving this issue open until I can post a link to the blog, for completeness. Thanks for the help, Gordon and Eddie!

**Comment by xjqian** [comment]
Ben, 

To answer your SSFP-like effect question, which I found interesting that we are now sensitive enough to be susceptible to these subtle issues, there is no addition gradient spoiling added to the Siemens product sequence in the CMRR MB sequences. However, there are three type of gradients in the Siemens product sequence related to that effect in an in vivo fMRI protocol with fat saturation:

1. The last readout lobe of the EPI is not refocused
2. There are a pair of matched gradients around the fat sat pulse. They add a bit of diffusion weighting for the water signal but doesn't add to gradient spoiling.
3. There is an additional gradient (oblique angle with X, Y, Z axis on) at the end of the fat sat module that serves additional gradient spoiling for water signal. However it is not as big as that proposed in the Zhao et al., 2000, MRM paper.

If additional gradient spoiling is indeed deemed desirable, I can imagine to reduce the gradient lobes before the fat sat pulse as a fairly simple modification. Nevertheless, some empirical evidences are needed before Eddie is willing to merge this to the mainline CMRR MB code. 

btw, as of R012 release, there is an "RF spoiling" option in the CMRR MB that could offer additional suppression of the SSFP-like effect in CSF.

**Comment by BenInglis** [comment]
Hi Gordon,

    I've not had time to investigate the SSFP possibilities in depth, but that is now on the docket since I've been able to demonstrate to my own satisfaction an absence of new, or significantly greater, motion sensitivity in MB-EPI than for product EPI with matched parameters. There's a full report in this blog post:

https://practicalfmri.blogspot.com/2016/10/respiratory-oscillations-in-epi-and-sms.html

I hope to add something more quantitative, via motion parameters (from realignment) in the coming days. But as of now I think we can safely close this issue and then, if my subsequent tests reveal any suspicion of SSFP effects at short TR, we can open a new issue. If you agree please go ahead and close this issue.

Thanks again for your help and suggestions. Changing the slice orientation was key to understanding what is happening.
